### DISCUSSION TITLE
除了VLESS-XTLS-Vision以外，其他配置是否“安全”？

### DISCUSSION THREAD

**Discussion by HuengchI** [lxhao61/integrated-examples#152] [Q&A] [answered]
感谢提供这个超级棒的仓库！

我刚接触这个“领域”，通过阅读大家的一些讨论，我得出结论：除了“VLESS-XTLS-Vision”（以及更新的[REALITY](https://github.com/XTLS/REALITY#readme)以外），别的配置组合都有一些安全问题。譬如，有些“不能跑大流量”，有些“不能在移动端使用”。我还没有彻底弄明白其中原理。

我发仓库中有许多配置，应该如何选择呢？他们都“安全”（自由地使用而被墙的风险很小）吗？

**Comment by lxhao61** [comment, marked as answer]
首先你要明白“安全”是相对与动态的，墙一直进化中。
目前 REALITY Vision 配置是最安全的，大于等于 XTLS Vision 配置；其次是固定 TLSv1.2 后使用非 AES 算法密码套件配置，再其次是 REALITY 配置，最后是常规配置。
当前选择参考如上说明，另外建议基于WebSocket 非 REALITY 应用仅套 CDN 使用。
